CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ message(" SRCNAME = ${SRCNAME} -> LIBNAME = ${LIBNAME}")
 option(build_python_module "Compile Python Wrappers" OFF)
 option(gtest_build_tests "Build Unit Tests" OFF)
 
-project(${LIBNAME} LANGUAGES C)
+project(${LIBNAME} VERSION 1.0 LANGUAGES C)
```

Comment:
I believe we're at 1.4
Will this be overwritten from what ImageStreamIO.h says, or will this value be the decisive version number?

Reply:
Edit: we're at 2.0, and ImageStruct.h is the file that says so.

Reply:
Ah, I missed that a different version was specified, thanks for pointing that out. I've updated the CMakeLists.txt to read the version from the ImageStruct.h macro.

Reply:
Edit: replaced that with a system similar to milk.